File: Exercises/answer_keys/cpp/mmc.cpp
#include <iostream>

using namespace std;

int mmc(int a, int b){
    int maior, menor = 0;
    
    if(a > b){
        maior = a;
        menor = b;
    }else{
        maior = b;
        menor = a;
    }

    while(maior%menor != 0){
        maior += maior;
    }

    return maior;
}

int main(){
    int a, b = 0;

    cout << "Input 2 numbers (separated by 'Enter'): " << endl;
    cin >> a; 
    cin >> b;

    cout << "The MMC between " << a << " and " << b << " is " << mmc(a, b) << endl;

    return 0;
}

File: Exercises/answer_keys/python/mmc.py
def mmc (a, b):
    
    if a > b:
        maior = a
        tmp = a
        menor = b
    else:
        maior = b
        tmp = b
        menor = a

    while (maior%menor != 0):
        maior += tmp

    return maior

def main():

    a, b = input ("Give me 2 numbers: ").split()
    a = int (a)
    b = int(b)

    print("The MMC between ", a, " and ", b, " is ", mmc(a, b))

if __name__ == "__main__":
    main()